src/shellCommands/ocSdkCommands.ts

```diff
@@ -267,7 +268,22 @@ export class OcSdkCommand {
     process.env.ANSIBLE_JINJA2_NATIVE = "true";
     const cmd: string = "ansible-playbook";
     args = args.concat("ibm.operator_collection_sdk.create_operator");
-    return this.run(cmd, args, outputChannel, logPath);
+
+    let commandOutput = "";
```

Comment:
I was worried that if we made this a mutable global variable and had a function to change it, we could potentially run in to race conditions if two operations try to change it at the same time. But since these commands are technically bound by `session.operationPending` I can make that change.

Reply:
I meant this block of code could be a common function that returns the promise 
![Screenshot 2024-02-01 at 1 43 34 PM](https://github.com/IBM/operator-collection-sdk-vscode-extension/assets/23127680/ff0df22e-fa68-40c0-90e8-718558d7cadf)